website/src/components/User/GroupPage.tsx

```diff
@@ -194,10 +208,58 @@ const InnerGroupPage: FC<GroupPageProps> = ({
                     </div>
                 </>
             )}
+
+            <div className=' max-w-2xl mx-auto px-10 py-4 bg-gray-100 rounded-md my-4'>
```

Comment:
could we still move this in front of the `Enter new user name` section?

Reply:
It's prettier now but without scrolling down you don't discover that this page allows you to add users so I do think we should get on with a redesign fairly soon, but happy to go with this for now